### PR TITLE
Fatal Zero formating

### PR DESCRIPTION
**Pull request by pgert**

### CODE REVIEW

File: patches/SLES-50821_22E91837.pnach
[the remark this answers is not in the document]
Comment (pgert) [reply patches/SLES-50821_22E91837.pnach:1]: Only the hack with "gsinterlacemode=1" should be activated by default if NI is enabled globaly. The other hack might confuse but someone might prefer it, so I included it.

File: patches/SLES-50821_22E91837.pnach [hunk 1]
```diff
@@ -27,76 +19,101 @@ patch=1,EE,00135B80,word,3C0141A8 // 3C0141C0
...
+author=ported by br4zil from NTSC-U
+patch=1,EE,00350F1C,word,00000000
+
+[Disable Post-Process and Noise effect - menus]
+author=ported by br4zil from NTSC-U
+comment=Makes main menu bland looking
+patch=1,EE,0035905C,word,00000000
+
+[Cheat - Infinite Health]
+patch=1,EE,20256824,extended,000001FD
```
[the remark this answers is not in the document]
Comment (pgert) [reply patches/SLES-50821_22E91837.pnach:116]: Well, why not?
Other hacks such as "Remove Blackbars" are "allowed",
 and this is only one hack - not a whole sheet of them.

[the remark this answers is not in the document]
Comment (refractionpcsx2) [reply patches/SLES-50821_22E91837.pnach:116]: Removing black bars are fine, cheats are not. Get rid of cheats.

File: patches/SCPS-56008_BCF04BF3.pnach [hunk 2]
```diff
@@ -1,44 +1,50 @@
...
 
-//Lens Flare's fix
-//Modified based on PAL (0,875 instead of 0,75) - better.
-patch=1,EE,00137154,word,3C013F12 //3c013f00
-patch=1,EE,00135b10,word,3C01428C //3c0142a0
-patch=1,EE,00135bf4,word,3C01410C //3c014120
-patch=1,EE,00135bbc,word,3C014128 //3c014140
-patch=1,EE,00135b78,word,3C0141A8 //3c0141c0
+// Lens Flare's fix
+// Modified based on PAL (0,875 instead of 0,75) - better.
```
Comment (refractionpcsx2) [inline patches/SCPS-56008_BCF04BF3.pnach:24]: whitespace

Comment (pgert) [reply patches/SCPS-56008_BCF04BF3.pnach]: Is it OK now?

Comment (pgert) [reply patches/SCPS-56008_BCF04BF3.pnach]: I guess so.

**Comment by Mrlinkwii** [comment]
@pgert  can you rebase the pr for me :)

**Comment by pgert** [comment]
You mean "squash" - No I can't.

Regarding the conflicts - What am I supposed to do?
Click on "Mark as resolved"?

**Comment by pgert** [comment]
Does this do?
If not, I have to make a new PR.
GitHub Desktop is beyond me.

**Review by F0bes — APPROVED** [review]
Yeah, that fixed the conflicts

**Pull request merged**